openmc/data/decay.py

```diff
@@ -298,6 +298,8 @@ class Decay(EqualityMixin):
         applications.
     decay_constant : uncertainties.UFloat
         Decay constant in inverse seconds.
+    decay_energy : float
+        Average energy in [eV] per decay for a decay heat
```

Comment:
`for a decay heat`? Isotope that decays?

Reply:
Oops, that was supposed to be "decay heat applications". Will fix that..